fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -82,12 +85,12 @@
     private static final String SEARCH_PARAMETER_MODIFIER_NAME =
             "Search parameter: '%s' must have resource type name modifier";
     private static final String INVALID_TARGET_TYPE_EXCEPTION = "Invalid target type for the Inclusion Parameter.";
-    private static final String UNSUPPOTED_EXPR_NULL =
+    private static final String UNSUPPORTED_EXPR_NULL =
             "An empty expression is found or the parameter type is unsupported [%s][%s]";
-    
+
     private static final String MODIFIYERRESOURCETYPE_NOT_ALLOWED_FOR_RESOURCETYPE =
             "Modifier resource type [%s] is not allowed for search parameter [%s] of resource type [%s].";
-    
+
     private static final String Different_MODIFIYERRESOURCETYPES_FOUND_FOR_RESOURCETYPES =
```

Comment:
While you are in here... can you refactor/rename to all upper case?

Reply:
done